### PR TITLE
Updated files under packaging with dependencies on tomcat8 to tomcat9.

### PR DESCRIPTION
**Pull request by cdelpinogeo**
Updated files under packaging with dependencies on tomcat8 to tomcat9.

### CODE REVIEW

File: packaging/ubuntu/14/depends.esv [hunk 1]
```diff
@@ -27,10 +27,10 @@ boundless-server-gs-sqlserver=boundless-server-geoserver (>= CURRENT_VER), bound
...
 boundless-server-gs-wps=boundless-server-geoserver (>= CURRENT_VER), boundless-server-geoserver (<< NEXT_VER)
 boundless-server-composer=boundless-server-geoserver (>= CURRENT_VER), boundless-server-geoserver (<< NEXT_VER)
-boundless-server-dashboard=boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6)
-boundless-server-docs=boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6)
-boundless-server-geoserver=unzip, boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6), libjpeg-turbo-official (>= 1.4.2)
-boundless-server-geowebcache=boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6)
+boundless-server-dashboard=boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6)
+boundless-server-docs=boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6)
+boundless-server-geoserver=unzip, boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6), libjpeg-turbo-official (>= 1.4.2)
+boundless-server-geowebcache=boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6)
```
Comment (tbarsballe) [inline packaging/ubuntu/14/depends.esv]: The versions in brackets will need to be updated

File: packaging/ubuntu/16/depends.esv [hunk 2]
```diff
@@ -27,10 +27,10 @@ boundless-server-gs-sqlserver=boundless-server-geoserver (>= CURRENT_VER), bound
...
 boundless-server-gs-wps=boundless-server-geoserver (>= CURRENT_VER), boundless-server-geoserver (<< NEXT_VER)
 boundless-server-composer=boundless-server-geoserver (>= CURRENT_VER), boundless-server-geoserver (<< NEXT_VER)
-boundless-server-dashboard=boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6)
-boundless-server-docs=boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6)
-boundless-server-geoserver=unzip, boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6), libjpeg-turbo-official (>= 1.4.2)
-boundless-server-geowebcache=boundless-server-tomcat8 (>= 8.5.32), boundless-server-tomcat8 (<< 8.6)
+boundless-server-dashboard=boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6)
+boundless-server-docs=boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6)
+boundless-server-geoserver=unzip, boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6), libjpeg-turbo-official (>= 1.4.2)
+boundless-server-geowebcache=boundless-server-tomcat9 (>= 8.5.32), boundless-server-tomcat9 (<< 8.6)
```
Comment (tbarsballe) [inline packaging/ubuntu/16/depends.esv]: Versions need to be updated

Comment (cdelpinogeo) [reply packaging/ubuntu/16/depends.esv]: Updated

**Comment by cdelpinogeo** [comment]
Pushed updated changes.

**Pull request merged**